content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -837,6 +837,30 @@ checking if the Job has the annotation
 this annotation from Jobs. Instead, you can recreate the Jobs to ensure they
 are tracked using Pod finalizers.
 
+### Elastic Indexed Jobs
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+{{< note >}}
+In order to use this behavior, you must enable the `ElasticIndexedJob`
```

Comment:
If the feature is on by default, then stating that you need to enable it is redundant, imo.